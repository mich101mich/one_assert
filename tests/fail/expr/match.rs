fn main() {
    one_assert::assert!(match 0 {
        1 => true,
        x => x + 1,
    });
}
