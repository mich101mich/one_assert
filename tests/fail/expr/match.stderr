error[E0308]: mismatched types
 --> tests/fail/expr/match.rs:4:14
  |
4 |         x => x + 1,
  |              ^^^^^ expected `bool`, found integer
